src/_locales/en/messages.json

```
@@ -807,6 +807,12 @@
   "environmentSaved": {
     "message": "The environment URLs have been saved."
   },
+  "disableKonnectorsSuggestions": {
+    "message": "Disable automatic connector suggestion"
+  },
+  "disableKonnectorsSuggestionsDesc": {
+    "message": "If a login matches a non-installed connector, suggest it in the Cozy."
```

```suggestion
    "message": "Prevent suggestions for logins that match a non-installed connector in the Cozy."
```

I think this fits better now that the option is "disable".